lib/liquid/combinators/tags/custom_tag.ex

```diff
@@ -0,0 +1,141 @@
+defmodule Liquid.Combinators.Tags.CustomTag do
+  import NimbleParsec
+  alias Liquid.Combinators.General
+
```

Comment:
Can you add a `@type t` spec for the Custom.t() type

Reply:
Done.